package.json

```diff
@@ -109,7 +109,7 @@
     "mongo:dev": "run-rs -v 5.0.23 -l ubuntu1804 --keep --dbpath mongodb-data --number 1 --quiet",
     "postinstall": "git config --global url.\"https://\".insteadOf git:// && gulp build && cd website/client && npm install",
     "apidoc": "gulp apidoc",
-    "heroku-postbuild": "npm run client:build"
```

Comment:
Do we actually no longer need `run client:build` ? :thinking:

Reply:
it was already run during the post install. The post build one was a duplicate that isn't actually needed. So yes, can remove this